data/presets/presets/morgue.json

```diff
@@ -0,0 +1,18 @@
+{
+    "icon": "maki-cemetery",
+    "fields": [
+        "religion",
+        "denomination"
+    ],
+    "geometry": [
+        "point",
+        "area"
+    ],
+    "terms": [
+        "memorial home"
+    ],
+    "tags": {
+        "amenity": "morgue"
```

Comment:
The tag for this is still `amenity=mortuary` even though I requested the label be "Morgue".